internal/pipeline/evaluator_test.go

```diff
@@ -27,10 +28,23 @@ import (
 	"github.com/spf13/afero"
 )
 
-func checkoutRepoStub(_ string, _ bool, _ *git.CloneOptions) (*git.Repository, error) {
+func checkoutRepoMock(_ string, _ bool, _ *git.CloneOptions) (*git.Repository, error) {
 	return &git.Repository{}, nil
 }
 
+func checkoutRepoFailureMock(_ string, _ bool, _ *git.CloneOptions) (*git.Repository, error) {
+	return nil, fmt.Errorf("unable to checkout repo")
+}
+
+func createWorkDirMock(_ afero.Fs, _ string, _ string) (string, error) {
+	_ = utils.AppFS.MkdirAll("/tmp/ec-work-1234", 0755)
+	return "/tmp/ec-work-1234", nil
```

Comment:
Would this be an issue if we hit this concurrently? Perhaps we could use [TempDir](https://pkg.go.dev/github.com/spf13/afero#TempDir) instead?

Reply:
This shouldn't be an issue as when this is called, we've set `utils.AppFS` to a new `afero.NewMemMapFs()` each time, see [this](https://github.com/hacbs-contract/ec-cli/pull/30/files/a3ed3a2775183376d0af696df63f65e941b034d6#diff-d16fab1c28968f90e4d9a19026cc0068615f5a92d49dc093d6d53096b78d67dcR316) as an example.